core/arch/arm/kernel/thread_a64.S

```diff
@@ -225,7 +227,7 @@ END_FUNC thread_unwind_user_mode
 	.endm
 
 #define INV_INSN	0
-FUNC thread_excp_vect , : align=2048
+FUNC thread_excp_vect , :, default, 2048 , naked
```

Comment:
Didn't
```
FUNC thread_excp_vect , : align=2048, attribute=naked
```
work?

Reply:
This also works. I will make the change.

Reply:
s/works/compiles but doesn't have the desired affect.

With this changed, in created tee.dmp, a new section named attribute=naked gets created instead of parameter attribute getting assigned with value naked.